apigee-cf-service-broker/api/service_instances.js

```diff
@@ -120,6 +120,32 @@ function actionValidate(req, res, next) {
   }
 }
 
+function deriveProtocol(params){
+  //default to https if no protocol is provided
+  const proto = params.protocol ? params.protocol.toString().trim().toLowerCase() : 'https'
+  const ret = {
+    errors: []
+  }
+  if (proto === "http" || proto === "https"){
+    ret.protocol = proto
+  }
+  else if (params.protocol){
+    ret.errors.push(params.protocol.toString().split(/[\s,]+/))
```

Comment:
If you `push` the input string to create a single-element array, then the way the error message is generated, it will just read like `bad protocol: "XYZ". Should be "http" or "https"`. Or you can change it to just be a string, and change the error message creation.